src/test/java/CollectionTest/SetTest.java

```diff
@@ -0,0 +1,42 @@
+package CollectionTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    public void test1() {
+        System.out.println(numbers.size());
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "3"})
+    void test2(String input) {
+        assertThat(numbers.contains(Integer.parseInt(input))).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    void test3(String input) {
```

Comment:
```suggestion
    void test3(int number, boolean result) {
```

4, 5의 경우 결과 값이 달라지기 때문에 위와 같이 구현해야 하지 않을까?